File: 🟢 Easy/263 - Ugly Number/_263UglyNumber.py
class Solution:
    def isUgly(self, n: int) -> bool:
        if n <= 0:
            return False

        primeFactors = [2, 3, 5]

        for i in range(0, 3):
            while n % primeFactors[i] == 0:
                n /= primeFactors[i]

        return n == 1

File: 🟢 Easy/263 - Ugly Number/_263UglyNumber.js
/**
 * @param {number} n
 * @return {boolean}
 */
var isUgly = function (n) {
    if (n <= 0) {
        return false;
    }

    let primeFactors = [2, 3, 5];
    for (let prime of primeFactors) {
        while (n % prime == 0) {
            n /= prime;
        }
    }

    // if 1 occurs then it is "UGLY NUMBER".
    return n == 1;
};